web/src/app/components/survey-list/_survey-list.component-theme.scss

```diff
@@ -5,28 +5,31 @@
   // TODO: Use theme colors once GM3 is supported (or custom palette is defined)
   // The colors defined below are a part of the GM3 color schemes, while the
   // theme colors are based on M2 palettes.
-  .surveys-page {
-    background-color: #edeee9; /* M3/sys/light/surface-container */
-  }
 
-  .surveys-page-title {
-    color: #1a1c19; /* M3/sys/light/on-surface */
+  .page {
+    background-color: #edeee9;
   }
 
-  .survey-card {
-    background-color: #f9faf4; /* M3/sys/light/surface */
-  }
+  .mat-mdc-card {
+    &.survey-card {
+      background-color: #f9faf4; /* M3/sys/light/surface */
```

Comment:
Are there SCSS variables we can refer to here and below instead of hard-coding the colors?

Reply:
I left it as they already are because this TODO comment:  

  // TODO: Use theme colors once GM3 is supported (or custom palette is defined)
  // The colors defined below are a part of the GM3 color schemes, while the
  // theme colors are based on M2 palettes.